x-pack/plugins/siem/public/store/timeline/epic.ts

```diff
@@ -236,6 +237,9 @@ export const createTimelineEpic = <State>(): Epic<
                         ...savedTimeline,
                         savedObjectId: response.timeline.savedObjectId,
                         version: response.timeline.version,
+                        timelineType: response.timeline.timelineType ?? TimelineType.default,
+                        templateTimelineId: response.timeline.templateTimelineId ?? null,
+                        templateTimelineVersion: response.timeline.templateTimelineVersion ?? null,
                         isSaving: false,
```

Comment:
I put these back as after updating/creating the timeline, we need to assign the value back to the store from api's response. Otherwise we might encounter with error like timeline's version conflict